catalog/transformer/catalog-transformer-nitf/src/test/java/ddf/catalog/transformer/nitf/TestNitfInputTransformer.java

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
Are we thinking about adding an integration test as well? We could use some additional Content Framework itests.